frankDoc/src/test/resources/doc/examplesExpected/simple.json

```diff
@@ -138,6 +135,18 @@
         "description":"If defined and empty or false, then this element and all its children are ignored"
       }]
     }
-  ]
+  ],
+  "enums": [{
+    "name": "nl.nn.adapterframework.frankdoc.testtarget.examples.simple.MyEnum",
+    "values": [
+      {
+        "label": "ONE"
+      },
+      {
+        "label": "customLabelTwo",
+        "description": "Description of TWO"
+      }
+    ]
+  }]
```

Comment:
Ik weet niet hoe handig het is dit te normaliseren. Enums worden maar 1x gebruikt, en dit zorgt voor extra complexiteit in de frontend.

Reply:
Deze pull request neemt de frontend mee. De complexiteit valt mee, denk ik.